### PR TITLE
fixing null workspace

### LINKED ISSUE #2656: null workspace creation
Steps to reproduce:
1. Load: https://remix.ethereum.org/#url=https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/contracts/app/solidity/mode.sol
2. then load: https://remix.ethereum.org/#address=0xdac17f958d2ee523a2206206994597c13d831ec7
3.  And again load: https://remix.ethereum.org/#url=https://github.com/ethereum/remix-project/blob/master/apps/remix-ide/contracts/app/solidity/mode.sol

check if there is a `null` workspace
Screenshot 2022-07-20 at 1 41 44 PM

### PR DESCRIPTION
**Pull request by LianaHus**
fix #2656

### CODE REVIEW

**Comment by yann300** [comment]
what shall I do to reproduce?

File: apps/remix-ide/src/app/panels/file-panel.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ module.exports = class Filepanel extends ViewPlugin {
     this.slitherHandle = new SlitherHandle()
     this.workspaces = []
     this.appManager = appManager
-    this.currentWorkspaceMetadata = {}
+    this.currentWorkspaceMetadata = null
```
Comment (ioedeveloper) [inline apps/remix-ide/src/app/panels/file-panel.js:60]: Is this change needed?

**Review by ryestew — COMMENTED** [review]
On Remix live - when I put swap out the github.com from this URL of a .sol file, it works fine:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

so this URL:
remix.ethereum.org/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

But if I use the raw file's url:
https://raw.githubusercontent.com/OpenZeppelin/openzeppelin-contracts/master/contracts/proxy/ERC1967/ERC1967Proxy.sol
so like this 
remix.ethereum.org/OpenZeppelin/openzeppelin-contracts/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

the file does not load and I get a null workspace.

Now on your PR - using this url - it should work but it does not load the github file:
http://localhost:8080/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

But at least the raw version does not give the null workspace:
localhost:8080/OpenZeppelin/openzeppelin-contracts/master/contracts/proxy/ERC1967/ERC1967Proxy.sol

File: apps/remix-ide/src/app/panels/file-panel.js (hunk 1 above)
Comment (LianaHus) [reply apps/remix-ide/src/app/panels/file-panel.js:60]: fartther you check if if it is null but it is an empty object

**Review by ryestew — APPROVED** [review]

**Pull request merged**